roles/network_plugin/calico/tasks/check.yml

```diff
@@ -37,6 +43,22 @@
   when:
     - "calico_vxlan_mode in ['Always', 'CrossSubnet']"
 
+- name: "Check Calico version minimum 3.19 when using eBPF dataplane"
+  assert:
+    that:
+      - "calico_version is version('v3.19.0','>=')"
+    msg: "you should use calico 3.19.0 or newer with eBPF dataplane"
```

Comment:
Just question:
According to https://www.projectcalico.org/introducing-the-calico-ebpf-dataplane/ which is mentioned in this pr, eBPF dataplane is implemented since Calico v3.13 as tech review.
Is that GA since v3.19?

Reply:
No, this is just from some discussions I had with project calico staff on their slack. I actually did the development for this PR on 3.18.3 and it worked OK but their recommendation was to use 3.19.0 as they would not invest into debugging older releases.

We can relax this completely but I would need to test with all the older versions to confirm behavior.

Reply:
I've tested 3.16.10, 3.17.4 and 3.18.3 and I was able to do successful deployments with all 3 versions. As such I removed this dependency and dropped the 3.19.0 change from this PR in favor of opening a separate one for the new calico version: https://github.com/kubernetes-sigs/kubespray/pull/7630